Microsoft.Toolkit.Wpf.UI.XamlHost/WindowsXamlHostBase.cs

```diff
@@ -64,28 +53,11 @@ public abstract partial class WindowsXamlHostBase : HwndHost
         /// </remarks>
         public WindowsXamlHostBase()
         {
-            // Windows.UI.Xaml.Application object is required for loading custom control metadata.  If a custom
-            // Application object is not provided by the application, the host control will create one (XamlApplication).
-            // Instantiation of the application object must occur before creating the DesktopWindowXamlSource instance.
-            // If no Application object is created before DesktopWindowXamlSource is created, DesktopWindowXamlSource
-            // will create a generic Application object unable to load custom UWP XAML metadata.
-            Microsoft.Toolkit.Win32.UI.XamlHost.XamlApplication.GetOrCreateXamlApplicationInstance(ref _application);
-
-            // Create an instance of the WindowsXamlManager. This initializes and holds a
```

Comment:
It would be helpful to put some comments describing the relationship of WindowsXamlManager to DesktopWindowsXamlSource somewhere and to describe scenarios where it would be explicitly used.